src/api/queries.js

```diff
@@ -34,17 +42,17 @@ exports.getQuestionnaire = `
               ...answerFragment
               ... on BasicAnswer {
                 secondaryLabel
+                parentAnswerId
```

Comment:
`parentAnswerId` isn't a field that can be queried against the API. See [BasicAnswer](https://github.com/ONSdigital/eq-author-graphql-schema/blob/master/index.js#L62) definition.